addons/website_livechat/static/src/embed/shadow.scss

```diff
@@ -0,0 +1,5 @@
+$website-color: map-get($colors, "o-color-1");
+
+.btn-outline-primary {
+  @include button-outline-variant($website-color);
```

Comment:
> The Chatbot's UI elements, including question selection boxes, should automatically adapt to the website theme.

I thin that the current implementation is too naive and superficial: 
- we only apply the theme color to the outline buttons. 
- if we were to change the button style (e.g., switch to filled instead of outline), this would no longer work.

It’s also inconsistent with other UI elements, such as the "continue" button and highlights (e.g. the search input in the emoji picker).

Wouldn’t it be better to update Bootstrap’s primary/secondary colors based on the website theme instead?

Also, I wonder if we shouldn't adapt chat window color according to the website theme? Either always sync it with the theme, or sync it if the value wasn't provided by the user?